101/check-setup.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
+
+cd $HOME
+
+tgt=$(vespa config get target -c never 2>&1)
+if [ "$tgt" != "target = cloud" ]; then
+	msg="Configure the CLI: ${tgt}"
+	echo "Problem:" "$msg"
+	exit 1
+fi
+
+app=$(vespa config get application -c never 2>&1)
+if [ "$app" = "application = <unset>" ]; then
+	msg="Configure the CLI: ${app}"
```

Comment:
Maybe same thing here? We should show what they should run?

Reply:
agreed, will make some changes to wording.